### DISCUSSION TITLE
I need some help with customizing a certain message with towny

### DISCUSSION THREAD

**Discussion by Blixy123** [TownyAdvanced/Towny#4949] [Q&A] [answered]
![Minecraft 1 16 5 - Multiplayer (3rd-party Server) 5_22_2021 10_12_18 AM](https://user-images.githubusercontent.com/84638688/119236109-78fea500-baea-11eb-952c-52185f1118a4.png)
I need some help in customizing/changing these messages, I know I can because I saw it on another server. 
![Minecraft 1 16 5 - Multiplayer (3rd-party Server) 5_22_2021 10_12_36 AM](https://user-images.githubusercontent.com/84638688/119236121-89168480-baea-11eb-914d-3994c37534d5.png)

**Comment by LlmDl** [comment, marked as answer]
The Towny config.yml's Notifications section is where all of these are coloured/formatted.
